requirements-periodic-scraping.txt

```diff
@@ -1,10 +0,0 @@
-attrs==17.3.0
```

Comment:
It would be better to squash this commit with the first

Reply:
I'm squashing all the commits on merge anyway. Do you still want me to squash it here?